README.md

```diff
@@ -115,6 +115,22 @@ Please note:
 7) [Optional] Disallow create/update on your roles or permissions after you define them, using the config file in **config/backpack/permissionmanager.php**. Please note permissions and roles are referenced in code using their name. If you let your admins edit these strings and they do, your permission and role checks will stop working.
 
 
+## Customize UserCrudController
+
+If you would like to add more fields to the default user controller provided by this package can bind your new controller to overwrite the one provided in this package:
+
+```php
+// in some ServiceProvider, AppServiceProvider for example
+
+$this->app->bind(
+        \Backpack\PermissionManager\app\Http\Controllers\UserCrudController::class, //this is package controller
+        \App\Http\Controllers\Admin\MyCrudController::class //this should be your own controller
+);
+
+// this tells Laravel that when UserCrudController is requested, your own CrudController should be served.
```

Comment:
```suggestion
// this tells Laravel that when UserCrudController is requested, your own UserCrudController should be served.
```

Reply:
@pxpm what do you think about suggesting users the correct name for their `UserCrudController`?
I'm not sure about it, but I think everyone understands we can name it what we want, we just suggest the most correct approach.